django-verdant/student_profiles/forms.py

```diff
@@ -221,12 +222,16 @@ def clean_image(self):
         if self.max_size and img.size > self.max_size:
             raise forms.ValidationError(u'Please keep file size under 10MB. Current file size {}'.format(filesizeformat(img.size)))
 
+        ALLOWED_FILE_TYPES = (
+            "image/jpeg",
+            "image/gif",
+            "image/png",
+        )
+
         try:
-            dt = Image.open(img)
-            dt.load()
-            if dt.format.upper() not in ('PNG', 'JPEG', 'GIF', 'MPO'):
+            file_type = magic.from_buffer(img.read(), mime=True)
+            if not file_type in ALLOWED_FILE_TYPES:
                 raise forms.ValidationError(u'Only images of types JPEG and GIF are allowed. Please make sure that you save the image file as the specified format, instead of simply changing the file extension.')
```

Comment:
The error message says only JPEG and GIF are allowed, but PNG is also in the allowed file type list.

Reply:
Ah yes, thanks will change that just now.